VERSIONING.md

```diff
@@ -76,32 +76,31 @@ Project versions should only be increased if a breaking change is introduced in
 for more details on plugin versioning.
 
-**NOTE:** the scheme for project version `"2"` was defined before the concept of plugins was introduced, so plugin `go.kubebuilder.io/v2` is implicitly used for those project types. Schema for project versions `"3-alpha"` and beyond define a `layout` key that informs the plugin system of which plugin to use.
+**NOTE:** the scheme for project version `"2"` was defined before the concept of plugins was introduced, so plugin `go.kubebuilder.io/v2` is implicitly used for those project types. Schema for project versions `"3"` and beyond define a `layout` key that informs the plugin system of which plugin to use.
 
 ## Introducing changes to plugins
 
 Changes made to plugins only require a plugin version increase if and only if a change is made to a plugin
 that breaks projects scaffolded with the previous plugin version. Once a plugin version `vX` is stabilized (it doesn't
 have an "alpha" or "beta" suffix), a new plugin package should be created containing a new plugin with version
-`v(X+1)-alpha`. Typically this is done by (semantically) `cp -r pkg/plugin/vX pkg/plugin/v(X+1)` then updating
+`v(X+1)-alpha`. Typically this is done by (semantically) `cp -r pkg/plugins/golang/vX pkg/plugins/golang/v(X+1)` then updating
 version numbers and paths. All further breaking changes to the plugin should be made in this package; the `vX`
 plugin would then be frozen to breaking changes.
 
+You must also add a migration guide to the [migrations](https://book.kubebuilder.io/migrations.html)
+section of the KubeBuilder book in your PR. It should detail the steps required
+for users to upgrade their projects from `vX` to `v(X+1)-alpha`.
+
 ### Example
 
-KubeBuilder scaffolds projects with plugin `go.kubebuilder.io/v2` by default. A `v3-alpha` version
-was created after `v2` stabilized.
+KubeBuilder scaffolds projects with plugin `go.kubebuilder.io/v3` by default.
 
 You create a feature that adds a new marker to the file `main.go` scaffolded by `init`
 that `create api` will use to update that file. The changes introduced in your feature
 would cause errors if used with projects built with plugins `go.kubebuilder.io/v2`
 without users manually updating their projects. Thus, your changes introduce a breaking change
 to plugin `go.kubebuilder.io`, and can only be merged into plugin version `v3-alpha`.
-This plugin's package should exist already, so a PR must be made against the
-
-You must also add a migration guide to the [migrations](https://book.kubebuilder.io/migrations.html)
-section of the KubeBuilder book in your PR. It should detail the steps required
-for users to upgrade their projects from `v2` to `v3-alpha`.
```

Comment:
Here I think we need to only replace v3-alpha for v3. We will have the migration steps after that as well.

Reply:
I placed that part before the example so that it doesn't look as part of the example and replaced `v2` and `v3-alpha` with `vX` and `v(X+1)-alpha`.